hugo.yaml

```diff
@@ -103,8 +103,14 @@ params:
   example_alpine_version: "3.19"
   example_node_version: "20"
 
-  min_api_threshold: 1.41
-
+  min_version_thresholds:
```

Comment:
Is it n-2 based? (n being the latest stable)

Reply:
ah, these are arbitrary atm so we need to decide. n-2 could be a good default. It might depend what the threshold is depending on the component, e.g. for Desktop n-2 might be too aggressive.